### PR TITLE
Add basic network inspection info

### PR DESCRIPTION
**Pull request by mheon**
Expose IP, port, and network namespace path via inspect

### CODE REVIEW

File: libpod/container_inspect.go [hunk 1]
```diff
@@ -52,8 +54,45 @@ func (c *Container) getContainerInspectData(size bool, driverData *driver.Data)
...
 	}
+
+	// Get information on the container's network namespace (if present)
+	if runtimeInfo.NetNS != nil {
+		// Get IP address
+		ip, err := c.runtime.getContainerIP(c)
+		if err != nil {
+			logrus.Errorf("error getting container %q IP: %v", config.ID, err)
+		}
+		data.NetworkSettings.IPAddress = ip.To4().String()
```
Comment (rhatdan) [inline libpod/container_inspect.go]: You can not do this if you have an error.

**Comment by rhatdan** [comment]
LGTM

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by TomSweeneyRedHat** [comment]
LGTM, I'm not sure if we'll need the EndpointID or SandboxID, but I don't have a deep understanding of how/what they're used for.  I see Moby has the SandboxID, but not the EndpointID.

**Pull request closed without merging**